diesel/Cargo.toml

```diff
@@ -39,6 +39,7 @@ lint = ["clippy"]
 large-tables = []
 huge-tables = ["large-tables"]
 postgres = ["pq-sys"]
+skip-timestamp-helpers = []
```

Comment:
Adding a cockroach feature would be ideal.  But in this case having a feature the enables the use of triggers in the database is also a good idea. This is business logic being defined in the database. It violates the rules of clean architecture by creating coupling between the app and the db. It's also not portable to the sqlite3 and mysql/mariadb back ends.